perses/rjmc/geometry.py

```diff
@@ -616,7 +674,7 @@ def _normalize_torsion_proposal(self, atom, r, theta, bond_atom, angle_atom, tor
         involved_torsions = self._get_torsions(atoms_with_positions, atom)
 
         #get an array of [0,2pi)
-        phis = units.Quantity(np.linspace(0, 2.0*np.pi, n_divisions), unit=units.radians)
+        phis = units.Quantity(np.arange(0, 2.0*np.pi, n_divisions), unit=units.radians)
```

Comment:
Looks like there's a discrepancy in vector sizes now because the syntax for [`arange`] is different:
```
======================================================================
ERROR: Run the geometry engine a few times to make sure that it actually runs
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/miniconda/envs/_test/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/travis/miniconda/envs/_test/lib/python2.7/site-packages/perses-0.1-py2.7.egg/perses/tests/test_geometry_engine.py", line 92, in test_run_geometry_engine
    new_positions, logp_proposal = geometry_engine.propose(sm_top_proposal)
  File "/home/travis/miniconda/envs/_test/lib/python2.7/site-packages/perses-0.1-py2.7.egg/perses/rjmc/geometry.py", line 135, in propose
    phi_proposed, logp_phi = self._propose_torsion(atom, r_proposed, theta_proposed, bond_atom, angle_atom, torsion_atom, torsion, atoms_with_positions, new_positions, beta)
  File "/home/travis/miniconda/envs/_test/lib/python2.7/site-packages/perses-0.1-py2.7.egg/perses/rjmc/geometry.py", line 741, in _propose_torsion
    phi_idx = np.random.choice(range(len(phis)), p=np.exp(logp))
  File "mtrand.pyx", line 1099, in mtrand.RandomState.choice (numpy/random/mtrand/mtrand.c:10676)
ValueError: a and p must have same size
```
This should be
```python
phis = units.Quantity(np.arange(0, 2.0*np.pi, (2.0*np.pi)/n_divisions), unit=units.radians)
```